ui/src/util/messageUtil.ts

```diff
@@ -0,0 +1,65 @@
+import * as __ from 'lodash';
+import { _ } from '@splunk/ui-utils/i18n';
+import messageDict from '../constants/messageDict';
+
+/**
+ * @param {number} code  a int value.
```

Comment:
JSDoc comments can be removed

Reply:
as soon as we have TS types

Reply:
do you know JSDoc comments can be used as a replacement for Typescript? The almost whole Svelte framework has been rewritten from TS to JSDocs
https://github.com/sveltejs/svelte/blob/main/packages/svelte/src/internal/server/index.js

Reply:
oh thats nice, did not know about that one, thanks for sharing,
also i removed those comments.